lib/linguist/languages.yml

```diff
@@ -4757,6 +4757,13 @@ Thrift:
   - ".thrift"
   ace_mode: text
   language_id: 374
+Ting:
+  type: programming
+  color: "#939dad"
+  extensions:
+  - ".tn"
+  - ".bl"
+  language_id: 422
```

Comment:
Errrm, you'll have a much easier time with this PR if you follow the instructions in [CONTRIBUTING.md](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#adding-a-language) and that includes using the `language_id` that is automatically generated by `script/set-language-ids` - it _is_ correct and valid. 😉